tests/test_end_to_end.py

```diff
@@ -9,18 +9,33 @@
 
 def test_employee_pipeline(tmpdir):
     source = current_path / "fixture_files" / "employees.csv"
-    EmployeeReviewPipeline(source=source, working_dir=tmpdir).run()
+    department_source = current_path / "fixture_files" / "departments.csv"
+    pipeline = EmployeeReviewPipeline(source=source, working_dir=tmpdir)
+    pipeline.init_source('departments', department_source)
```

Comment:
If it always happens before run, I wonder why it's not part of the constructor.  But I don't feel strongly.